docusaurus/docs/operate/configs/envoy_config.md

```diff
@@ -208,3 +208,9 @@ endpoints:
       capacity_limit: 100000
       capacity_limit_period: "CAPACITY_LIMIT_PERIOD_MONTHLY"
 ```
+
+- [`.gateway-endpoints.yaml` example file](https://github.com/buildwithgrove/path-auth-data-server/blob/main/yaml/testdata/gateway-endpoints.example.yaml).
```

Comment:
Plese remove these. See how I update the docs at the top of the `### Gateway Endpoint Functionality` section.